code/modules/mob/living/carbon/human/species/shadow.dm

```diff
@@ -16,7 +17,7 @@
 		"brain" = /obj/item/organ/brain
 		)
 
-	flags = NO_BLOOD | NO_BREATHE | NO_SCAN
+	flags = NO_BLOOD | NO_BREATHE | NO_SCAN | NO_POISON
```

Comment:
This is the species you become if you use one of the slime toxins (it was either green or black, can't remember), although the shadowling species is a child of this one, it redefines its own flags.